File: app/backend/src/main/kotlin/at/htlleonding/maturaballmanager/model/dtos/PromDTO.kt
package at.htlleonding.maturaballmanager.model.dtos

data class DayPlanDTO(
    var name: String? = null,
    var time: String? = null
)

data class PromDTO(
    var motto: String? = null,
    var date: String? = null,
    var time: String? = null,
    var street: String? = null,
    var houseNumber: String? = null,
    var zip: String? = null,
    var city: String? = null,
    var dayPlan: List<DayPlanDTO>? = null
)

File: app/frontend/web/src/app/model/dtos/prom.dto.ts
export interface DayPlanDTO {
    name: string;
    time: string;
}

export interface PromDTO {
    motto: string;
    date: string;
    time: string;
    street: string;
    houseNumber: string;
    zip: string;
    city: string;
    dayPlan: DayPlanDTO[];
}
